pkg/autoscaler/config/config.go

```diff
@@ -248,6 +280,12 @@ func validate(lc *Config) (*Config, error) {
 		return nil, fmt.Errorf("panic-window-percentage = %v, must be in [%v, 100] interval", lc.PanicWindowPercentage, 100*float64(BucketSize)/float64(lc.StableWindow))
 	}
 
+	if lc.DefaultInitialScale < 0 {
+		return nil, fmt.Errorf("default-initial-scale = %v, must be greater than 0", lc.DefaultInitialScale)
```

Comment:
I think this should be:

```suggestion
		return nil, fmt.Errorf("default-initial-scale = %v, must be at least 0 (or at least 1 when allow-zero-initial-scale is false)", lc.DefaultInitialScale)
```